src/components/Root/FFetch.js

```diff
@@ -60,157 +70,65 @@ const OKAPI_FETCH_OPTIONS = {
 };
 
 export class FFetch {
-  constructor({ logger }) {
+  constructor({ logger, store }) {
     this.logger = logger;
-
-    // save a reference to fetch, and then reassign the global :scream:
-    this.nativeFetch = global.fetch;
-    global.fetch = this.ffetch;
-
-    this.NativeXHR = global.XMLHttpRequest;
-    global.XMLHttpRequest = FXHR(this);
+    this.store = store;
   }
 
-  /** { atExpires, rtExpires } both are JS millisecond timestamps */
-  tokenExpiration = null;
-
-  /** lock to indicate whether a rotation request is already in progress */
-  // @@ needs to be stored in localforage???
-  isRotating = false;
-
   /**
-   * isPermissibleRequest
-   * Some requests are always permissible, e.g. auth-n and forgot-password.
-   * Others are only permissible if the Access Token is still valid.
-   *
-   * @param {Request} req clone of the original event.request object
-   * @param {object} te token expiration shaped like { atExpires, rtExpires }
-   * @param {string} oUrl Okapi URL
-   * @returns boolean true if the AT is valid or the request is always permissible
+   * save a reference to fetch, and then reassign the global :scream:
    */
-  isPermissibleRequest = (resource, te, oUrl) => {
-    if (isValidAT(te, this.logger)) {
-      return true;
-    }
-
-    const isPermissibleResource = (string) => {
-      const permissible = [
-        '/bl-users/forgotten/password',
```

Comment:
Now that this whitelist is removed, are we still able to make requests when not authenticated to perform actions like reset password?

Reply:
Yes. Previously, we needed the allow-list because we inspected every non-allow-listed request to see if its AT was valid. Now, since RTR requests take place out of band from regular request, we no longer do that inspection, so we no longer need the allow-list.